boards/arm/efr32bg_sltb010a/efr32bg_sltb010a.dts

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (c) 2021 Sateesh Kotapati
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <silabs/efr32bg22c224f512im40.dtsi>
+
+/ {
+	model = "Silicon Labs EFR32BG SLTB010A (aka Thunderboard BG22)";
+	compatible = "silabs,efr32bg_sltb010a", "silabs,efr32bg22";
+
+	/* These aliases are provided for compatibility with samples */
+	aliases {
+		led0 = &led0;
+		sw0 = &button0;
+		watchdog0 = &wdog0;
+	};
+
+	chosen {
+		zephyr,console = &usart1;
+		zephyr,shell-uart = &usart1;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led0: led_0 {
+			gpios = <&gpiob 0 0>;
+			label = "LED 0";
+		};
+	};
+
+	buttons {
+		compatible = "gpio-keys";
+		button0: button_0 {
+			/* gpio flags need validation */
+			gpios = <&gpiob 1 GPIO_ACTIVE_LOW>;
+			label = "User Push Button 0";
+		};
+	};
+};
+
+&cpu0 {
+	clock-frequency = <76800000>;
```

Comment:
Use macro to define clock either in KHZ or MHZ

Reply:
Except I2C_BITRATE_STANDARD, I don't see cpu-frequency is using a macro. If we need to define a macro, we need to add it in dt-bindings include file.

Reply:
Oh. Sorry. I thought, it was clock configuration in the board. Actually cpu node does not belongs to board usually, but to soc instead. Take a look here: https://docs.zephyrproject.org/latest/guides/porting/board_porting.html

It could be that some work is already done for your soc.